letsencrypt/client/client.py

```diff
@@ -339,7 +339,10 @@ def init_key():
     the namedtuple to easily work with the protocol.
 
     """
-    key_pem = crypto_util.make_key(CONFIG.RSA_KEY_SIZE)
+    if key_size < CONFIG.RSA_KEY_SIZE:
+        logging.warning("Generating keys smaller than %d bits is NOT recommended!", CONFIG.RSA_KEY_SIZE)
+
+    key_pem = crypto_util.make_key(key_size)
```

Comment:
Crypto throws ValueError("RSA modulus length must be a multiple of 256 and >= 1024").  We should probably handle this error a bit more nicely.

Using keys weaker than 2048 bit RSA won't be allowed by any CA.
https://cabforum.org/wp-content/uploads/BRv1.2.3.pdf (Page 34)

Although there might be a benefit from allowing the client to work with internal unrestricted CAs???
I would recommend at least handling the Crypto exceptions nicely.  We will have to have a mechanic to increase the size of the key based on the CA error response... so leaving the ability for 1024 is probably OK.  Thoughts?